crates/storage/src/structured_storage/blocks.rs

```diff
@@ -47,6 +60,21 @@ impl TableWithBlueprint for FuelBlocks {
     }
 }
 
+impl TableWithBlueprint for FullFuelBlocks {
+    type Blueprint = Merklized<
```

Comment:
We don't need to create a Merkle tree over the blocks used only by P2P

Reply:
addressed in #2159

Reply:
precisely in https://github.com/FuelLabs/fuel-core/pull/2159/files#diff-0e430672a8d94fbbca929879774abd2f09dd2a86eca77c32c77615e100b51831R57-R64